src/libsync/determineauthtypejob2.cpp

```diff
@@ -0,0 +1,49 @@
+#include "determineauthtypejob2.h"
+
+#include "common/utility.h"
+#include "creds/httpcredentials.h"
+
+#include <QLoggingCategory>
+
+Q_LOGGING_CATEGORY(lcDetermineAuthTypeJob, "sync.networkjob.determineauthtype2", QtInfoMsg);
+
+using namespace OCC;
+
+DetermineAuthTypeJobFactory::DetermineAuthTypeJobFactory(QNetworkAccessManager *nam, QObject *parent)
+    : AbstractCoreJobFactory(nam, parent)
+{
+}
+
+DetermineAuthTypeJobFactory::~DetermineAuthTypeJobFactory() = default;
+
+Job *DetermineAuthTypeJobFactory::startJob(const QUrl &url)
+{
+    auto job = new Job;
+
+    QNetworkRequest req(Utility::concatUrlPath(url, QStringLiteral("remote.php/dav/files/")));
```

Comment:
TODO: which url to use with spaces

Reply:
Should I add such a TODO? Please don't hesitate to provide some text.